packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -66,7 +67,7 @@ export interface IMergeNodeCommon {
     isLeaf(): this is ISegment;
 }
 
-export type IMergeNode = IMergeBlock | ISegment;
+export type IMergeNode = IMergeBlock | IRootMergeBlock | ISegment;
```

Comment:
this shouldn't be necessary. goal is to not expose outside to the package if we can avoid it.